<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Exit</class>
 <widget class="QWidget" name="Exit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="PicExit">
   <property name="geometry">
    <rect>
     <x>-2</x>
     <y>-5</y>
     <width>1281</width>
     <height>731</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Pic/Exit.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="YesButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="NoButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>380</y>
     <width>111</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
